rust/otap-dataflow/crates/otap/Cargo.toml

```diff
@@ -79,11 +79,11 @@ rustls-native-certs = { workspace = true, optional = true }
 base64 = { workspace = true, optional = true }
 arc-swap = { workspace = true, optional = true }
 rustls = { workspace = true, optional = true }
-rustls-pemfile = { workspace = true, optional = true }
+rustls-pki-types = { workspace = true, optional = true }
```

Comment:
`rustls-pemfile` is no longer maintained, so replaced by `rustls-pki-types`, which is maintained by `rustls` project.